pkg/pool-manager/pod_pool.go

```diff
@@ -61,7 +61,7 @@ func (p *PoolManager) AllocatePodMac(pod *corev1.Pod) error {
 	}
 
 	// validate if the pod is related to kubevirt
-	if p.isRelatedToKubevirt(pod) {
+	if p.isRelatedToKubevirt(pod, networks) {
```

Comment:
Please elaborate in the commit message why do we need this change and what is the logic behind it.